Documentation/5.4.1/README.rst

```diff
@@ -438,11 +520,11 @@ For AVRS integration, follow these steps:
 
 :Step 2: When modifying the overcloud-full image", use the script provided to update the image. No additional changes are required.
 
-:Step 3: Create a new sriov-role.yaml file to deploy SR-IOV Compute nodes. The command used to create this file is:
+:Step 3: Create a new compute-sriov-role.yaml file to deploy SR-IOV Compute nodes. The command used to create this file is:
 
 ::
 
-    openstack overcloud roles generate Controller Compute ComputeSriov -o /home/stack/templates/sriov-role.yaml
+    openstack overcloud roles generate Controller Compute ComputeSriov -o /home/stack/nuage-tripleo-heat-templates/roles/compute-sriov-role.yaml
```

Comment:
Here we need to do 2 things, we have to make a copy of all roles from /usr/share/tht/roles to nuage-tripleo-heat-templates/roles/.
Then generate the role file by passing "--roles-path /home/stack/nuage-tripleo-heat-templates/roles/" and generate the role-file in /home/stack/templates/.
Because ideally /home/stack/nuage-tripleo-heat-templates/roles/ should only have all available roles but not role-files.